backend/src/main/java/heartbeat/util/StringToEnumConverter.java

```diff
@@ -1,6 +1,6 @@
 package heartbeat.util;
 
-import heartbeat.controller.board.vo.request.BoardType;
+import heartbeat.controller.board.dto.request.BoardType;
```

Comment:
this file `covert` method has been implamented by `heartbeat.controller.board.dto.request.BoardType`, 
PLZ refactor to remove `covert`